djangoevents/__init__.py

```diff
@@ -31,10 +34,19 @@
 
 
 def publish(event):
-    warnings.warn("`publish` is depreciated. Please switch to: `store_event`", DeprecationWarning)
+    warnings.warn("`publish` is depreciated. Please switch to: `store_event`.", DeprecationWarning)
     return es_publish(event)
 
 
 def store_event(event):
-    return es_publish(event)
+    """
+    Store an event to the service's event journal. Optionally validates event
+    schema if one is provided.
+    """
+    if schema_validation_enabled():
+        is_valid = validate_event(event)
+        if not is_valid:
+            msg = "Event: {} does not match its schema.".format(event)
+            raise EventSchemaError(msg)
```

Comment:
There is no error dictionary. Just True/False response unfortunately (this one really bugs me).